src/metabase/db/custom_migrations.clj

```diff
@@ -246,6 +246,54 @@
                                                           [:<> :result_metadata nil]
                                                           [:<> :result_metadata "[]"]]})))))
 
+(defn- remove-opts
+  "Removes the `join-alias`, `temporal-unit`, and `binning` options from the `field_ref` options map.
```

Comment:
Nit: you're not removing `join-alias` in one of the calls so this is slightly incorrect

Reply:
updated: [Update docstring](https://github.com/metabase/metabase/pull/27487/commits/7c57e9f352ff62359f41f1aeaec1c9c122e37d6c)